src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -155,6 +156,16 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
                 </th>
               ),
             )}
+            {actions && !actionsOnTheLeft &&
+              (actions.columnTitle != null ? (
```

Comment:
can we not duplicate the columns code here? Either make it into a separate component, function. Maybe we could also separate the markup from the data. `columns = [leftActions, ...columns, ...rightActions].map(... markup ...)`

Reply:
@fwermelinger Moved to function :)

Reply:
thanks